alphapept/interface.py

```
@@ -938,6 +975,8 @@ def run_complete_workflow(
         if workflow["recalibrate_data"]:
             steps.append(recalibrate_data)
             steps.append(search_data)
+        if workflow["search_glycans"]:
```

please rewrite this as `if workflow.get('search_glycans')` for backwards compatibility. The standard settings.yaml file will error out if the _search_glycans_ field is not present in it rather than skip the step.

---

not sur what you mean here. Can you update it and merge?